portal-web/test/functional/com/liferay/portalweb/paths/portlet/theme/Test3Theme.path

```diff
@@ -0,0 +1,19 @@
+<html>
+<head>
+<title>Test3Theme</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">Test3Theme</td></tr>
+</thead>
+<tbody>
+<tr>
+    <td>SCREENSHOTS_THEME_LAYOUT_PNG</td>
```

Comment:
The name of the path screenshot should reflect what it's checking. The current name implies that it's a general theme layout.

Something like TEST_THEME_7.3_HOME_PAGE_PNG would be better. And leave out the liferay dxp logo and text, we want the test to be flexible enough to test on CE bundles.

I'm wondering if there's a better portlet or area to check since the screenshot is only really checking for two elements: navigation and Hello World portlet title...